build/Dockerfile

```diff
@@ -1,27 +1,29 @@
-FROM quay.io/kubescape/ebpf-engine:v1.0.0 as builder
+FROM quay.io/kubescape/ebpf-engine:v1.0.0 as base
 
+FROM golang:1.20-alpine as builder
 
-FROM ubuntu:latest
+ENV GO111MODULE=on 
+ENV CGO_ENABLED=0
 
-RUN apt update && apt-get install git libelf-dev golang-go -y
+RUN mkdir /etc/node-agent
```

Comment:
why building in /etc?

Reply:
I dont know. This is what was in the previous Dockerfile and I do not know what dependencies we need from there.
We can have this sorted out in a different PR.

Reply:
ok, when we make the move to efficient multi-platform builds 👍